jkamz/Questioner
# #163071862 create new meetup record

### What does this PR do?
Creates and endpoint for creating meetups

### Description of the tasks to be completed
Create an endpoint allowing admins to create meetups

### How can this be manually tested?
Clone this repository on local machine and CD into project folder

`$ git clone https://github.com/jkamz/Questioner`

`$ cd Questioner`

Check out to the PR's branch
`$ git checkout ft-create-meetup-record-163071862`

Install app dependencies from requirements.txt

Run the following commands
`source venv/bin/activate`
`export FLASK_ENV=development`
`flask run`

Use postman to test the endpoint

### What are the relevant Pivotal Tracker stories?
#163071862

### Screenshots

![create meet post](https://user-images.githubusercontent.com/11965186/50957977-a4136b80-14d0-11e9-942d-25831eb8e728.png)
